### ISSUE TITLE
remove /etc/resolve.conf

### ISSUE THREAD

**Issue by ioerror** [subgraph/subgraph-os-issues#1]
The default system should never need to send unencrypted DNS requests. Any app, such as a browser for bypassing captive portals, may - and that should be in a container only.

**Comment by mckinney-subgraph** [comment]
resolvconf was initially added because Oz had a dependency on it. But we do need to control DNS is a way that forces it through Tor when we need to and sends it to static or DHCP-provided nameservers when we don't need to. So we're currently assessing how we want to do this.

As it currently stands, we have setup resolvconf so DNS defaults to `127.0.0.1` in the host (but resolvconf is a little buggy and sometimes will pull in DHCP-provided nameservers before 127.0.0.1 -- we plan to patch this). Oz also supports having a different `/etc/resolv.conf` in sandboxes for things like captive portal bypass, which currently uses resolvconf but doesn't need to obviously. 

In short, resolvconf will probably be removed in the future after we evaluate how we want to handle different DNS policies.

**Comment by ioerror** [comment]
I just put ```nameserver 127.0.0.1``` in ```/etc/resolve.conf``` and did ```chattr +i /etc/resolv.conf``` to keep it unchanged. I also did some reconfiguration for dhclient/NM but that is just hacky nonsense.
